scripts/library.sh

```diff
@@ -491,8 +491,12 @@ function get_latest_knative_yaml_source() {
     # Get the latest tag name for the current branch, which is likely formatted as v0.5.0
     local tag_name="$(git describe --tags --abbrev=0)"
     # The given repo might not have this tag, so we need to find its latest release manifest with the same major&minor version.
-    local major_minor="$(echo ${tag_name} | cut -d. -f-2)"
-    echo "$(gsutil ls gs://knative-releases/${repo_name}/previous/${major_minor}.*/${yaml_name}.yaml | tail -n 1)"
+    local major_minor="$(echo ${tag_name} | cut -d. -f1-2)"
+    local yaml_source_file="$(gsutil ls gs://knative-releases/${repo_name}/previous/${major_minor}.*/${yaml_name}.yaml \
+      | sort \
+      | tail -n 1 \
+      | sed 's/gs:\/\//https:\/\/storage.googleapis.com\//')"
```

Comment:
Use `cut -b6-` to remove the `gs://` prefix, then add the https and domain; this will be more readable than the see command.

Reply:
Done.